### PR TITLE
Support hdfs broker reading config from hdfs-site.xml

### PR DESCRIPTION
**Pull request by caiconghui**
In the production environment, hdfs broker should read config from hdfs-site.xml instead of  manual input due to the complexity of hdfs config
add dfs.namenode.kerberos.principal.pattern config for kerberos auth
fix some getting config code for brief

### CODE REVIEW

**Comment by imay** [comment]
@caiconghui 
Thank you for your contribution.
Could you please create an issue and relate this PR to it?

File: fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java [hunk 1]
```diff
@@ -140,22 +142,17 @@ public BrokerFileSystem getFileSystem(String path, Map<String, String> propertie
...
-            }
+        String username = properties.getOrDefault(USER_NAME_KEY, "");
+        String password = properties.getOrDefault(PASSWORD_KEY, "");
+        String dfsNameServices = properties.getOrDefault(DFS_NAMESERVICES_KEY, "");
+        String authentication = properties.getOrDefault(CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION,
+            AUTHENTICATION_SIMPLE);
+        if (Strings.isNullOrEmpty(authentication) || (!authentication.equals(AUTHENTICATION_SIMPLE)
+            && !authentication.equals(AUTHENTICATION_KERBEROS))) {
+            logger.warn("invalid authentication:" + authentication);
+            throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
```
Comment (kangpinghuang) [inline fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java]:
```suggestion
            throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
```

**Review by kangpinghuang — DISMISSED** [review]
Just need a small fix by removing the extra space.

**Comment by kangpinghuang** [comment]
LGTM

**Comment by caiconghui** [comment]
@imay  related issue https://github.com/apache/incubator-doris/issues/2151

**Comment by caiconghui** [comment]
@kangpinghuang  fix, done

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**